catalog-rest-service/src/main/java/org/openmetadata/catalog/security/DefaultAuthorizer.java

```diff
@@ -132,6 +153,7 @@ public boolean hasPermissions(AuthenticationContext ctx, EntityReference owner)
   public boolean hasPermissions(
       AuthenticationContext ctx, EntityReference entityReference, MetadataOperation operation) {
```

Comment:
Something to note here...

In this API, entityReference refers to the entity being updated



In the following API, we are using reference to owner of the entity 
```
public boolean hasPermissions(AuthenticationContext ctx, EntityReference owner) {
```